packages/smart/contracts/augur-para/IFeePot.sol

```diff
@@ -1,13 +1,14 @@
-pragma solidity 0.5.15;
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
 
 import "../libraries/IERC20.sol";
 import "../libraries/IERC20DynamicSymbol.sol";
 
 
-contract IFeePot is IERC20 {
-    function depositFees(uint256 _amount) external returns (bool);
-    function withdrawableFeesOf(address _owner) external view returns(uint256);
-    function redeem() external returns (bool);
-    function collateral() external view returns (IERC20);
-    function reputationToken() external view returns (IERC20DynamicSymbol);
+abstract contract IFeePot is IERC20 {
```

Comment:
We originally wrote these like abstract contracts but they ought to be rewritten as interfaces. We used `contract` only because solidity 5's interface support was far too limiting.